lab_submission.sql

```diff
@@ -1 +1,9 @@
 -- CREATE EVN_average_customer_waiting_time_every_1_hour
+CREATE EVENT EVN_average_customer_waiting_time_every_1_hour
+ON SCHEDULE EVERY 1 HOUR
+DO
+  INSERT INTO customer_service_kpi (customer_service_KPI_average_waiting_time_minutes)
```

Comment:
The code requires several adjustments for it to run. It is not correct as it is.